grails-app/taglib/au/org/ala/biocache/hubs/OccurrenceTagLib.groovy

```diff
@@ -538,13 +538,88 @@ class OccurrenceTagLib {
         }
     }
 
+    /**
+     * Provides custom formatting per occurrence property (key).
+     *
+     * @attr value REQUIRED
+     * @attr key REQUIRED
+     * @return formatted value
+     */
+    def formatValue = { value, key ->
+        if(key == 'eventDate') {
+            return g.formatDate(date: new Date(value), format:"yyyy-MM-dd")
+        } else if(key == 'collectors' && value.size() > 2) {
+            return value[0..1].join(', ') + ', ...'
+        } else if(key == 'collectors') {
+            return value[0..-1].join(', ') // XXX funny
```

Comment:
I think it's nicer "if key == 'collectors' " is checked once and then value.size() check is nested inside it